android/app/build.gradle

```diff
@@ -39,6 +39,8 @@ android {
         targetSdkVersion 30
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
+        // The number of method references in a .dex file cannot exceed 64K
+        multiDexEnabled true
```

Comment:
Why we need to enable this?

Reply:
Because the number of method references in a .dex file exceed 64K

Reply:
Ok, noted to improve